pkgs/servers/akkoma/admin-fe/default.nix

```diff
@@ -2,26 +2,25 @@
 , stdenv
 , fetchFromGitea, fetchYarnDeps
 , fixup-yarn-lock, yarn, nodejs
+, git
 , python3, pkg-config, libsass
 }:
 
 stdenv.mkDerivation rec {
   pname = "admin-fe";
-  version = "unstable-2024-02-25";
+  version = "stable-2024-04-27";
```

Comment:
The package naming guidelines recommend using an `unstable` prefix for packages not based on an explicit upstream release: <https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming>